packages/admin/cms-admin/src/blocks/image/EditImageDialog.tsx

```diff
@@ -149,7 +159,7 @@ export function EditImageDialog({ image, initialValues, onSubmit, onClose, inher
                             <div>
                                 {inheritedDamSettings !== undefined && (
                                     <>
-                                        <Box padding={8}>
+                                        <Box padding={8} paddingBottom={6}>
```

Comment:
Does it look alright if the dependencies aren't configured? Or should the padding be added to the box below?

Reply:
![image](https://github.com/user-attachments/assets/7e9821a6-2ba2-4629-ae13-c4f0e0fc3251)

Looks good to me